src/utils/helpers.ts

```diff
@@ -20,7 +20,7 @@ const truncateHash = (revision: RevisionsList['revision']) =>
 // return only most recent commit message
 // first commit is usually 'try_task_config'
 const getLatestCommitMessage = (item: RevisionsList) =>
-  item.revisions[item.revisions.length - 1].comments;
+  item.revisions[0].comments;
```

Comment:
We probably need to be smarter than that: the concern in the comment is still present on try builds, because the last commit is usually the try commit.

Can we know whether an item has only a change for `try_task_config.json`? Maybe we need some help from the server from this?
Alternatively, we could use the selected repository (that would be passed to the function): if it's "try" then take index 1 instead of 0.

Reply:
@julienw I don't have a lot of context about this, do you know who can I reach out to for clarifying some of your questions?